NuRadioReco/eventbrowser/dataprovider_root.py

```diff
@@ -0,0 +1,26 @@
+import NuRadioReco.modules.io.rno_g.rnogDataReader
+
+
+class DataProviderRoot(object):
+    __instance = None
+
+    def __new__(cls):
+        if DataProviderRoot.__instance is None:
+            DataProviderRoot.__instance = object.__new__(cls)
+        return DataProviderRoot.__instance
+
+    def __init__(self):
+        self.__user_instances = {}
+
+    def get_arianna_io(self, user_id, filename):
```

Comment:
Somehow this seems like this is mixing RNO-G and ARIANNA now ... not sure I completely understand.

Reply:
We inherited this from the previous eventbrowser - the file access function was called get_arianna_io. It should be relatively straightforward to rename it to something more neutral everywhere, e.g. get_io_handler

Reply:
Thanks for fixing this. This has actually been around for way too long. The files were called .ari, before we decieded for something more general as .nur